### PR TITLE
Fixes #18286

### PR DESCRIPTION
**Pull request by amousset**

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/common/1.0/cf-serverd.st [hunk 1]
```diff
@@ -155,9 +155,13 @@ bundle common def
...
 body server control
 {
+      policy_server::
         trustkeysfrom     => {
           @{def.acl} ,
-        }; #trustkey allows the exchange of keys
+        };
+      # our policy server should already be trusted at bootstraping
+      !policy_server::
+        trustkeysfrom     => {};
```
Comment (ncharles) [inline techniques/system/common/1.0/cf-serverd.st:165]:
```suggestion
    any::
```

Comment (amousset) [reply techniques/system/common/1.0/cf-serverd.st:165]: YES

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/common/1.0/cf-serverd.st [hunk 2]
```diff
@@ -155,10 +155,15 @@ bundle common def
 
 body server control
 {
+      policy_server::
         trustkeysfrom     => {
           @{def.acl} ,
-        }; #trustkey allows the exchange of keys
+        };
+      # our policy server should already be trusted at bootstraping
```
Comment (peckpeck) [inline techniques/system/common/1.0/cf-serverd.st:162]: at bootstraping -> after bootstraping is done

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/common/1.0/update.st [hunk 3]
```diff
@@ -428,3 +451,10 @@ body perms u_mog(mode,user,group)
   owners => { "${user}" };
   groups => { "${group}" };
 }
+
+
+bundle edit_line rudder_update_lines_present(lines)
```
Comment (ncharles) [inline techniques/system/common/1.0/update.st:456]: all other bodys/bundle have a more specific name. I fear collision with a name that common
Could you name it u_rudder_update_lines_present ?

Comment (peckpeck) [reply techniques/system/common/1.0/update.st:456]: Where is this used

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**